Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -69,58 +81,98 @@ public void Validate()
         #region Options
 
         /// <summary>
-        /// Hostname of the server to connect to
+        /// Hostname of the server that the client will connect to.
         /// </summary>
         public string Host { get; set; } = DEFAULT_HOST;
 
         /// <summary>
-        /// Port of the server to connect to
+        /// Port of the server that the client will connect to.
         /// </summary>
         public ushort Port { get; set; } = DEFAULT_PORT;
 
         /// <summary>
-        /// The font family to use on misc non-ui rendering
+        /// The font family to use on misc non-ui rendering.
         /// </summary>
         public string GameFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on entity names
+        /// The font family to use on entity names.
         /// </summary>
         public string EntityNameFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on chat bubbles
+        /// The font family to use on chat bubbles.
         /// </summary>
         public string ChatBubbleFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on action messages
+        /// The font family to use on action messages.
         /// </summary>
         public string ActionMsgFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on unstyled windows such as the debug menu/admin window
+        /// The font family to use on un-styled windows such as the debug menu/admin window.
         /// </summary>
         public string UIFont { get; set; } = DEFAULT_UI_FONT;
 
         /// <summary>
-        /// Number of lines to save for chat scrollback
+        /// Number of lines to save for chat scrollback.
         /// </summary>
         public int ChatLines { get; set; } = DEFAULT_CHAT_LINES;
 
         /// <summary>
-        /// Menu music file name
+        /// Sets a music file to be played in the main menu.
         /// </summary>
         public string MenuMusic { get; set; } = DEFAULT_MENU_MUSIC;
 
         /// <summary>
-        /// Menu background art
+        /// Sets a list from which introductory images will be drawn when the game client starts.
+        /// </summary>
+        public List<string> IntroImages { get; set; } = new List<string>();
+
+        /// <summary>
+        /// Sets a texture to be used as static background in the main menu.
         /// </summary>
         public string MenuBackground { get; set; } = DEFAULT_MENU_BACKGROUND;
 
-        // TODO: What is this for?
-        public List<string> IntroImages { get; set; } = new List<string>();
+        /// <summary>
+        /// Toggles the ability to stretch the static background in the main menu.
+        /// </summary>
+        public bool MenuBackgroundStretched { get; set; } = DEFAULT_MENUBG_STRETCH;
```

Comment:
No. No. No. No. No.

This should be an enum of multiple display modes. If you only want to write the implementation for the default (center screen), and stretched, that's fine -- but it absolutely should not be a boolean.

These are all display modes that would be ideal to support, boolean is not good enough (and honestly stretching if the aspect ratio is different is hideous)

- Center screen
- Center horizontally
- Center vertically
- Fit (letter/pillarboxed)
- Cover (the smaller dimension fits, the larger one hangs off the edge of the screen)
- Stretched
- Tiled

Reply:
decided to add the following for now:
![imagen](https://user-images.githubusercontent.com/17498701/149366804-8f3998c7-0090-4ec9-8b05-583be6f2ef49.png)

Reply:
Created the following Enum for now:

![imagen](https://user-images.githubusercontent.com/17498701/149393214-fd7a0791-69f1-41f6-a65d-952ced93ba93.png)